### PR TITLE
Fixes/app store mobile

### PR DESCRIPTION
**Pull request by zomars**
## What does this PR do?

- Adds missing tabs in mobile
- Fixes App Store grid in mobile 
- Fixes grid in App Store categories in mobile

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

- [ ] Go to `/apps` in mobile
- [ ] Go to `/apps/installed` in mobile

### CODE REVIEW

File: apps/web/components/apps/AllApps.tsx [hunk 1]
```diff
@@ -9,7 +9,7 @@ export default function AllApps({ apps }: { apps: App[] }) {
   return (
     <div className="mb-16">
       <h2 className="mb-2 text-lg font-semibold text-gray-900">{t("all_apps")}</h2>
-      <div className="grid-col-1 grid gap-3 md:grid-cols-3">
+      <div className="grid-col-1 grid grid-cols-1 gap-3 md:grid-cols-3">
```
Comment (zomars) [inline apps/web/components/apps/AllApps.tsx:12]: There was a missing breakpoint for mobile in here.

File: apps/web/pages/apps/categories/[category].tsx [hunk 2]
```diff
@@ -28,7 +28,7 @@ export default function Apps({ appStore }: InferGetStaticPropsType<typeof getSta
         </div>
         <div className="mb-16">
           <h2 className="mb-2 text-lg font-semibold text-gray-900">All {router.query.category} apps</h2>
-          <div className="grid grid-cols-3 gap-3">
+          <div className="grid-col-1 grid grid-cols-1 gap-3 md:grid-cols-3">
```
Comment (zomars) [inline apps/web/pages/apps/categories/[category].tsx:31]: Matched with the main listing.

File: apps/web/pages/apps/installed.tsx [hunk 3]
```diff
@@ -307,13 +308,15 @@ export default function IntegrationsPage() {
...
   return (
     <Shell heading={t("installed_apps")} subtitle={t("manage_your_connected_apps")}>
-      <ClientSuspense fallback={<Loader />}>
-        <IntegrationsContainer />
-        <CalendarListContainer />
-        <WebhookListContainer title={t("webhooks")} subtitle={t("receive_cal_meeting_data")} />
-        <IframeEmbedContainer />
-        <Web3Container />
-      </ClientSuspense>
+      <AppsShell>
```
Comment (zomars) [inline apps/web/pages/apps/installed.tsx:311]: Added Missing Tabs in installed page

**Review by sean-brydon — APPROVED** [review]
Looks much better on mobile :) LGTM

**Pull request merged**